alf/algorithms/algorithm.py

```diff
@@ -889,7 +889,7 @@ def get_initial_train_state(self, batch_size):
         return r
 
     @common.add_method(nn.Module)
-    def state_dict(self, destination=None, prefix='', visited=None):
+    def state_dict(self, destination=None, prefix='', visited=None, **kwargs):
```

Comment:
Maybe add a short comment explaining why kwargs is needed here.

Reply:
> Maybe add a short comment explaining why kwargs is needed here.

added